tests/unit/test_charm.py

```diff
@@ -257,6 +257,51 @@ def test_gateway_minio_missing_args(harness):
     harness.begin_with_initial_hooks()
 
     assert harness.charm.model.unit.status == BlockedStatus(
-        "Minio in gateway mode requires gateway_storage_service configuration. "
+        "Minio in gateway mode requires gateway-storage-service configuration. "
         "Possible values: s3, azure"
     )
+
+
+def test_gateway_minio_with_private_endpoint(harness):
```

Comment:
I would like to see another test for when gateway is enabled but the config values are missing.

Reply:
Would that not be the `test_gateway_minio_args` test?

Reply:
Okay nevermind, just saw that you had added that one before. All good in that case.